### PR TITLE
docs(nuxt): deploy to vercel guide

### PR DESCRIPTION
**Pull request by mandarini**
A guide explaining how you can deploy a Nuxt application to Vercel.

### CODE REVIEW

File: docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+# Deploying Nuxt applications to Vercel
+
+## Configure your Nuxt application appropriately for Vercel deployment
+
+In your application's `nuxt.config.ts` file, ensure that the `nitro` property is configured to use the `vercel` preset, and that the `output.dir` property is set to the appropriate directory. The directory that Vercel expects to find the application's build output (as explained in the [Vercel documentation](https://vercel.com/docs/build-output-api/v3)) in is `.vercel/output` at the root of the repository/workspace. So, if your application is located at `apps/my-app` then the `output.dir` property should be set to `../../.vercel/output`.
+
+For example, if your Nx workspace is structured like this:
+
+```treeview
```
Comment (jaysoo) [inline docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md:14]: Normally the treeview is the result of running the `tree .` command. Like

```treeview
├── apps
│   └── demo
│       └── .babelrc
├── libs
│   ├── a
│   │   └── .babelrc
│   └── b
│       └── .babelrc
└── babel.config.json
```

File: docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+## Configure your Vercel project's settings appropriately
+
+### New Vercel project
+
+1. If you are "importing" your Nx workspace's repository for the first time, make sure you do _not_ choose a root directory as part of the repo selection process (therefore leaving it to be the root of the full repo/workspace)
+2. Ensure the Nuxt.js "Framework Preset" is selected
+3. Expand the "Build & Development Settings" and toggle the override switch for the build command. For example, for an application named `my-app` the value will look like this:
+
+```shell
+npx nx build my-app --prod
```
Comment (jaysoo) [inline docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md:84]: I don't think we need `--prod` here, the build is defaulted to production config. (we also need to update the Next.js deploy docs separately)

**Comment by jaysoo** [comment]
For the screenshots I'm not sure if we have a best practice to show dark or light mode.

File: docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md (hunk 1 above)
Comment (mandarini) [reply docs/shared/recipes/deployment/deploy-nuxt-to-vercel.md:14]: oops!

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**